### PR TITLE
CFE-3524/master: Added ability to specify additional directories to add autorun policy from

### PR DESCRIPTION
**Pull request by nickanderson**
With this change, when services_autorun is defined and if
def.mpf_additional_auto_input_dirs is defined then policy files (*.cf) in those
directories (if they exist as directories) will be automatically added to
inputs.

Ticket: CFE-3524
Changelog: Title

### CODE REVIEW

**Comment by bim9262** [comment]
@nickanderson, I was able to verify that our custom autorun files were picked up!

**Comment by nickanderson** [comment]
thanks @bim9262

**Comment by nickanderson** [comment]
@olehermanse didn't you have some bikeshedding you wanted to do on this variable name?

**Comment by nickanderson** [comment]
@olehermanse variables renamed as you suggested.

@bim9262 please note changes here.

File: MPF.md [hunk 1]
```diff
@@ -632,12 +632,12 @@ defined. Bundles tagged with ```autorun``` are **not required** to be placed in
 
 #### Additional automatically loaded inputs
 
-When `def.mpf_additional_auto_input_dirs` is defined (and services_autorun is defined), the policy files (`*.cf`) in those directories will be added to inputs. If a directory is specified but is not a directory, it will be skipped.
+When `def.mpf_extra_autorun_inputs` is defined (and services_autorun is defined), the policy files (`*.cf`) in those directories will be added to inputs. If a directory is specified but is not a directory, it will be skipped.
```
Comment (olehermanse) [inline MPF.md:635]: I think there should be a link here for how to add `.cf` policy files to inputs.

File: promises.cf.in [hunk 2]
```diff
@@ -331,11 +331,30 @@ bundle common cfengine_controls
 
 bundle common services_autorun
 # @brief Include autorun policy and discover autorun bundles if enabled
+# Directories defined in `def.mpf_extra_autorun_inputs` will have their
+# inputs automatically added as well (if the directory exists, if not it will be
+# silently skipped).
```
Comment (olehermanse) [inline promises.cf.in:336]: > Directories will have their inputs automatically added

Directories will have their files added to inputs automatically

Or 

Files inside directories listed in `def.mpf_extra_autorun_inputs` will be added to inputs automatically.

**Review by olehermanse — CHANGES_REQUESTED** [review]
LGTM, please squash.

File: promises.cf.in (hunk 2 above)
Comment (nickanderson) [reply promises.cf.in]: ACK

File: MPF.md (hunk 1 above)
Comment (nickanderson) [reply MPF.md:635]: ACK

**Review by olehermanse — APPROVED** [review]

**Pull request merged**